### PR TITLE
docs:add weekly report 20230202

### PR DESCRIPTION
**Pull request by KID-G**
Changes:

Add  weekly report 20230202

### CODE REVIEW

File: blog/zh/blog/2023/02/02/weekly-report-0202.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    Apache APISIX Ingress 1.6 版本正式发布！此版本带来了 Gateway API 的支持，同时新增服务发现组件集成与代理外部服务，优化对 APISIX Ingress Controller 的使用体验。
+
+- [2022 Apache APISIX 年度记忆](https://apisix.apache.org/zh/blog/2023/01/02/2022-summary/)
+
+    本文将带您回顾 Apache APISIX 社区在 2022 年在各领域取得的成绩。
+
+- [Apache APISIX 3.1.0 正式发布](https://apisix.apache.org/zh/blog/2022/12/30/release-apache-apisix-3.1.0/)
+
+    Apache APISIX 3.1.0 版本正式发布！此版本带来很多关于安全层面的功能支持，同时新增内置调试插件，旨在优化对 APISIX 的使用体验。
```
Comment (yzeng25) [inline blog/zh/blog/2023/02/02/weekly-report-0202.md:63]: These blogs are mentioned in the previous bi-weekly report. Do you really want to repeat them here?

Comment (KID-G) [reply blog/zh/blog/2023/02/02/weekly-report-0202.md:63]: Not exactly, I also updated some of the links

Comment (yzeng25) [reply blog/zh/blog/2023/02/02/weekly-report-0202.md:63]: ok

**Review by yzeng25 — APPROVED** [review]
LGTM

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**